.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,5 +2,10 @@ Fixes #<issue_number_goes_here>
 
 > It's a good idea to open an issue first for discussion.
 
-- [ ] Tests pass
-- [ ] Appropriate changes to README are included in PR
\ No newline at end of file
+-   [ ] Local tests pass (running `run_desktop_tests.sh`)
+-   [ ] Tested against boards
```

Comment:
A solution outside the scope of this PR would be to add a compiler flag for a "test mode". This test mode removes all user presence checks, like I currently do in my [remote debugging branch](https://github.com/kaczmarczyck/OpenSK/tree/remote-debugging). Then you just run the Test Tool against all board in test mode. This should cover all your use cases.

Downsides are that it takes a few minutes per board in wall time. Also, the Test Tool would need a touchless mode that omits all tests that would fail. Therefore, a short term solution would be to just ask to check webauthn.io or have a short Python script that sends a register and login?